api/src/main/java/org/pmiops/workbench/api/OfflineUserController.java

```diff
@@ -201,4 +209,26 @@ private boolean timestampsEqual(Timestamp a, Timestamp b) {
     }
     return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
   }
+
+  /**
+   * Audits GCP access for all users in the database.
+   *
+   * This API method is called by a cron job and is not part of our normal user-facing surface.
+   */
+  @Override
+  public ResponseEntity<Void> bulkAuditProjectAccess() {
```

Comment:
We're missing a test for this method – see examples to draw from in OfflineUserControllerTest.java. Don't need to go overboard, but having nonzero test coverage is important.